game-of-life-multiplatform/settings.gradle

```diff
@@ -1,3 +1,4 @@
 rootProject.name = 'game-of-life'
 
-include 'common', 'common-jvm', 'common-js', 'terminal-jvm', ':android', 'web', ':desktop-jvm'
+include 'common', 'common-jvm', 'common-js', 'terminal-jvm', ':android', 'web', ':desktop-jvm', 'iOS'
```

Comment:
`nit` For consistency sake, I think we should have `:ios` here, also small case. What do you think?